content/en/docs/tasks/administer-cluster/verify-signed-images.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Verify Signed Container Images
+content_type: task 
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+
+## Pre-requisites
+
+- [cosign](https://docs.sigstore.dev/cosign/installation/)
+
+## Verifying image signatures
+For a complete list of images that are signed please refer to [releases](/releases/download/) page.
+
+Let's pick one image from this list and verify its signature
+using `cosign verify` command:
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+
+{{% alert title="Note" %}}
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+this [page](https://github.com/sigstore/cosign/blob/main/KEYLESS.md)
+{{% /alert %}}
+
+### All Control Plane Images
+
+To verify all signed control plane images, please run this command:
+
+```shell
+curl https://kubernetes.io/examples/admin/signed-images/auto-generated-list-of-all-signed-images.txt --output auto-generated-list-of-all-signed-images.txt
+input=auto-generated-list-of-all-signed-images.txt
+while IFS= read -r image
+do
+  COSIGN_EXPERIMENTAL=1 cosign verify "$image"
+done < "$input"
+```
```

Comment:
The list of signed images should be a release artifact and not stored as an example configuration.

Reply:
@sftim Maybe this could go in here: https://github.com/kubernetes/website/tree/main/static under a new "rel-eng" directory? @saschagrunert what do you think?

Reply:
`rel-eng` is a team detail; we would not expose that to end users.

I'd like to keep SIG Docs away from signing and verification, leaving that as something that SIG Release look after. The places where those artifacts and signatures and lists of signatures belong should change less frequently than our website, and there should be more process attached to updating those.

The way I see it, the _Kubernetes website_ is not where we host artifacts, signatures or lists of signatures. If someone wants to verify a signed artifact, all of that process - bar the page that teaches them how - ought to be hosted separately from the website.

Reply:
Is kubernetes.dev a better fit for this page?

Reply:
To move this PR ahead, maybe we can host the text file under `content/en/docs/tasks/administer-cluster/`.
In the long run, this file should be generated and published at https://github.com/kubernetes/kubernetes/releases or somewhere.